tests/unit/test_tokens.py

```diff
@@ -9,6 +9,38 @@ def test_client_token(aau_token_path):
     assert exists(aau_token_path)
 
 
+def test_client_token_add_hostid(aau_token_path):
+    assert not exists(aau_token_path)
+    token1 = utils._random_token()
+    with open(aau_token_path, "w") as fp:
+        fp.write(token1)
+    token2 = tokens.client_token()
+    assert token1 == token2
+    with open(aau_token_path) as fp:
+        token3 = fp.read()
+    assert token3.split(" ", 1)[0] == token2, (token2, token3)
+    assert token3.split(" ", 1)[1] == utils._get_node_str(), token3
+    utils._cache_clear()
+    token4 = tokens.client_token()
+    assert token4 == token2, (token2, token4)
+
+
+def test_client_token_replace_hostid(aau_token_path):
```

Comment:
testing the scenario where the host ID changes. in this case, a new token should be created.